src/main/java/org/bonitasoft/connectors/rest/RESTConnector.java

```diff
@@ -536,7 +536,7 @@ public void execute(final Request request) throws Exception {
             requestBuilder.setUri(urlStr);
             setHeaders(requestBuilder, request.getHeaders());
             var method = HTTPMethod.valueOf(requestBuilder.getMethod());
-            if (HTTPMethod.GET != method && HTTPMethod.HEAD != method) {
+            if (HTTPMethod.GET != method && HTTPMethod.HEAD != method && HTTPMethod.DELETE != method) {
```

Comment:
I suggest defining a `hasBody()` abstract method.
And specializing it in the concrete classes.

What do you think ?